react/getting_started_with_react/keys_in_react.md

```diff
@@ -42,14 +42,14 @@ Keys are passed into the component or a DOM element as a prop. You should alread
 <div key={keyValue} />
 ```
 
-<span id="keys-from-data">Now that we know the syntax, the next question is: what should be used as a key? Ideally, they should be some identifier that is unique to each item in the list. Most databases assign a unique id to each entry, so you shouldn't have to worry about assigning an id yourself. If you are defining data yourself, it is good practice to assign a unique `id` to each item. You may use the [uuid package](https://www.npmjs.com/package/uuid) to generate a unique id. Let's look at an example:</span>
+<span id="keys-from-data">Now that we know the syntax, the next question is: what should be used as a key? Ideally, they should be some identifier that is unique to each item in the list. Most databases assign a unique id to each entry, so you shouldn't have to worry about assigning an id yourself. If you are defining data yourself, it is good practice to assign a unique `id` to each item. You can use the [`crypto.randomUUID()` function](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) to generate a unique id. Let's look at an example:</span>
```

Comment:
![image](https://github.com/user-attachments/assets/f86d5929-c2ba-4704-8953-efdadf5629a6)
Since we only have effectively two "words", one backticked and one not, I think the rendering looks a bit odd. Removing the backticks IMO makes it look nicer.
```suggestion
<span id="keys-from-data">Now that we know the syntax, the next question is: what should be used as a key? Ideally, they should be some identifier that is unique to each item in the list. Most databases assign a unique id to each entry, so you shouldn't have to worry about assigning an id yourself. If you are defining data yourself, it is good practice to assign a unique `id` to each item. You can use the [crypto.randomUUID() function](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID) to generate a unique id. Let's look at an example:</span>
```

Reply:
Yeah, I went back and forth on this one. Agree on a relook that its probably better without